### ISSUE TITLE
Feature: Cross-file training/validation set selector

### ISSUE THREAD

**Issue by ASAAR** [ASAAR/SE2-KaggleComp#6]
We have multiple files with the training data, so we need to have a script (R or Python) that selects our training/validation sets from across these files.

**Comment by ASAAR** [comment]
https://www.kaggle.com/c/bosch-production-line-performance/forums/t/22909/expeditive-exploration-models-on-data

**Comment by RyanMcBerg** [comment]
We also discussed a tool to merge files based on a variable, and a way to save data to a smaller filetype, as well as .CSV.  I will be researching to see if python has a binary type that is easily usable, and attempt to provide options based on the users preference.
